skimage/restoration/_rolling_ball.py

```diff
@@ -53,10 +53,19 @@ def rolling_ball(image, *, radius=100, kernel=None, nansafe=False, num_threads=N
     noise). If this is a problem in your image, you can apply mild
     gaussian smoothing before passing the image to this function.
 
+    This algorithm is polynomial in the radius, with exponent = image.ndim,
```

Comment:
```suggestion
    This algorithm's complexity is polynomial in the radius, and exponential with image dimensionality,
```

Hope I translated that correctly :eyes:

Reply:
Hmm.. my understanding was that it's always polynomial, not exponential, but the *degree* of the polynomial matches the number of image dimensions? Is that right, @jni?

Reply:
Assuming that is the case, I force-pushed b980cb95bb183d0357258582a7b142dcd6ce09e3 to change the wording accordingly.